docs/pages/reference/js/dscvr/getDscvrFrameMessage.mdx

```diff
@@ -0,0 +1,27 @@
+# getDscvrFrameMessage
+
+Returns a `DscvrFrameMessageReturnType` object from the message payload. Throws an error if the message is invalid. Requires the `@dscvr/frames-validator` peer dependency to be installed.
+
+## Usage
+
+```ts
+yarn add @dscvr/frames-adapter
+```
+
+```ts
+import { getDscvrFrameMessage } from "frames.js/dscvr";
```

Comment:
```suggestion
import { getDscvrFrameMessage } from "@dscvr-one/frames.js";
```

It'd be from the frames.js forked package right?

Reply:
Thanks, done